frontend/mulighetsrommet-veileder-flate/src/components/ikkeKvalifisertVarsel/BrukerKvalifisererIkkeVarsel.tsx

```diff
@@ -1,20 +1,22 @@
 import { Alert } from '@navikt/ds-react';
-import { useBrukerHarRettPaaTiltak } from '../../hooks/useBrukerHarRettPaaTiltak';
+import { Bruker, Innsatsgruppe } from 'mulighetsrommet-api-client';
 import appStyles from '../../App.module.scss';
 import styles from './BrukerKvalifisererIkkeVarsel.module.scss';
-import { Innsatsgruppe } from 'mulighetsrommet-api-client';
-import { useHentBrukerdata } from '../../core/api/queries/useHentBrukerdata';
 
-export function BrukerKvalifisererIkkeVarsel() {
-  const {
-    brukerHarRettPaaTiltak,
-    brukersInnsatsgruppe,
-    innsatsgruppeForGjennomforing = Innsatsgruppe.STANDARD_INNSATS,
-  } = useBrukerHarRettPaaTiltak();
-
-  const brukerdata = useHentBrukerdata();
+interface Props {
+  brukerdata?: Bruker;
+  brukerHarRettPaaTiltak: boolean;
+  brukersInnsatsgruppe?: Innsatsgruppe;
```

Comment:
Er `brukersInnsatsgruppe` noe annet enn `brukerdata.innsatsgruppe`? I så fall kunne man kanskje spart seg for denne

Reply:
Bra spottet! Ser det er samme verdi som kommer fra brukerdata, men fra to forskjellige hooks. Jeg fikser og forenkler.